src/CLU/Microsoft.CLU/CommandLineParser/UnixCommandLineParser.cs

```diff
@@ -121,11 +121,11 @@ public bool Parse(ICommandBinder commandBinder, string[] arguments)
                 }
                 else
                 {
-                    if (position == 0 && 
-                        commandBinder.SupportsAutomaticHelp && 
+                    if (position == 0 &&
+                        commandBinder.SupportsAutomaticHelp &&
                         arg.Equals("help", System.StringComparison.OrdinalIgnoreCase))
```

Comment:
+1 for idea for the method .. but I'm wondering if we should consider -h ... if -h is an optional parameter for any of the azure commands it would be confusing